packages/lib/src/io-command.ts

```diff
@@ -246,8 +255,8 @@ const inputFlags = {
 }
 
 /**
- * Mixin meant to be used along with Outputting and/or Listing that provides
- * instance of OutputOptions.
+ * Since both Outputting and Listing need OutputOptions, this is a simple
+ * mixin that needs to be included when one or both of those mixins are used.
```

Comment:
I usually try to use a doc format like `@see` when referencing symbols in comments in case we refactor later. However, I tried with VSCode and couldn't really get it to link to code or refactor with it as well as I have with IntelliJ. So our milage will very with htat.

Reply:
We can look into that more but I'm not sure that tsdoc supports it. I think we should consider something like Sphinx for our docs at some point anyway...I think that will play better with actually getting them posted somewhere and give us a lot more flexibility.